test/index.test.js

```diff
@@ -2,93 +2,120 @@ const sinon = require('sinon')
 const fs = require('fs-extra')
 const vault2Env = require('../index')
 
-describe('vault2env', function () {
```

Comment:
Oh, sorry - I removed `vault2env` label because it doesn't add anything useful...

Reply:
Use this link for review: https://github.com/kiwicom/vault2env-js/pull/4/files?w=1
It's more readable with parameter `w=1`.